src/components/DistanceRequest.js

```diff
@@ -137,19 +155,41 @@ function DistanceRequest({transactionID, transaction, translate}) {
                     innerStyles={[styles.ph10]}
                 />
             </View>
+            <View style={styles.mapViewContainer}>
+                {!isOffline && mapboxAccessToken.token ? (
```

Comment:
```suggestion
                {!isOffline && Boolean(_.get(mapboxAccessToken, 'token')) ? (
```

Should we use lodashGet ? and better to use `Boolean(mapboxAccessToken.token)`?